src/commcare_cloud/ansible/roles/datadog/templates/elastic.yaml.j2

```diff
@@ -7,3 +7,6 @@ instances:
   - url: http://{{ inventory_hostname }}:9200
     tags:
       - environment:{{ env_monitoring_id }}
+{% if elasticsearch_version is version('7.0.0', '>=') %}
+      - es_version: {{ elasticsearch_version }}
```

Comment:
any reason not to always include this tag?

Reply:
I guess we could, but this was mainly to identify the metrics just during the upgrade process.